onyo/lib/consts.py

```diff
@@ -46,3 +47,32 @@
 
 Use for type matching (:py:func:`onyo_get`) and output.
 """
+
+ONYO_DIR = Path('.onyo')
+r"""Path of the onyo directory.
```

Comment:
I like the original here. "The onyo directory" is very vague.

> The Path of the "dot onyo" directory that contains the onyo configuration, templates, etc.

Reply:
Hm. You commented on that, too. So, you did.

Changing back to original.